GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+particularly with respect to its strategic direction and long-term health, may
+be nominated to become a maintainer. This role is described below.
+
+## Maintainers 
+
+Maintainers are community members who have shown that they are committed to the
+continued development of the project through ongoing engagement with the
+community.
+
+Typically, a potential maintainer will need to show that they have an
+understanding of the project, its objectives and its strategy. They will also
+have provided valuable contributions to the project over a period of time.
+
+The project maintainers consists of those individuals identified as
+‘project owners’ on the development site. The maintainers have additional
+responsibilities over and above those of a contributor. These responsibilities
+ensure the smooth running of the project. Maintainers are expected to review
+code contributions, participate in strategic planning, approve changes to the
+governance model and manage the copyrights within the project outputs.
+
+Maintainers do not have significant authority over other members of the
+community, although they make decisions when community consensus cannot be
+reached. In addition, maintainers have access to the project’s private mailing
+list.  This list is used for sensitive issues, such as votes for new
+maintainers and legal matters that cannot be discussed in public. It is never
+used for project management or planning.
+
+Maintainer membership is by invitation from the existing members. A nomination
+will result in discussion and then a vote by the existing maintainers on the
+project’s private management list.
+Maintainer membership votes are subject to consensus approval of the current
+maintainer members.
+
+# Support
+
+All participants in the community are encouraged to provide support for new
+users within the project management infrastructure. This support is provided as
+a way of growing the community. Those seeking support should recognise that all
+support activity within the project is voluntary and is therefore provided as
+and when time allows. A user requiring guaranteed response times or results
+should therefore seek to purchase a support contract from a community member.
+However, for those willing to engage with the project on its own terms, and
+willing to help support other users, the community support channels are ideal.
+
+# Contribution process
+
+Anyone can contribute to the project, regardless of their skills, as there are
+many ways to contribute. For instance, a contributor might be active on the
+project mailing list and issue tracker, or might supply patches.
+
+# Decision making process
+
+Decisions about the future of the project are made through discussion with all
+members of the community, from the newest user to the most experienced
+maintainer member. All non-sensitive project management discussion takes place
+on the project contributors’ mailing list. Occasionally, sensitive discussion
```

Comment:
More mailing list references that we do not have.

Reply:
I need to change to the NM one (our current public devel one).
For the maintainers, as I mentioned before.

Reply:
Changed to `developer`.